vuepress/docs/next/tutorials/create/ms/run-local.md

```diff
@@ -3,178 +3,157 @@ sidebarDepth: 2
+1.  If you are already running a widget, click `Ctrl+C` to end the process
 
 2.  Go to the form widget folder in your project:
 
         cd ui/widgets/YOUR-ENTITY-NAME/formWidget
 
-3.  Then install and start your widget executing the command:
+3.  Install and start your widget with the command:
 
         npm install && npm start
 
-4.  When the widget is started a browser window is opened with and the widget URL is loaded
+4.  When the widget starts, a browser window opens, with the widget URL 
 
-5.  If you’re not logged in you’re redirected to the login page.
+5.  If you’re not logged in, it will redirect you to the login page.
 
 6.  Log in using:
 
         Username: user
         Password: user
 
-7.  You’ll be redirected to the widget page and you can see the widget form with the ID 1 loaded.
-
-#### Form widget notes:
-
-If you want to load other data you have to change the index.html file in the folder:
+7.  Once logged in, you will see the widget form.
 
-    cd ui/widgets/YOUR-ENTITY-NAME/formWidget/public
+#### Form Widget notes:
 
-and change the id attribute in this line:
+If you want to use custom data, edit the `index.html` file in the public folder ui/widgets/YOUR-ENTITY-NAME/detailsWidget/public. Change the "id" attribute in this line:
 
     <my-entity-form service-url="%REACT_APP_SERVICE_URL%" id="1" />
 
-### Start the details widget
+### Start the Details Widget
 
-You can also start your generated details widget:
+1. If you are already running a widget, click `Ctrl+C` to end the process
 
-1.  If you are running another widget, stop it clicking `Ctrl+C` in your widget command line window
-
-2.  Go to the details widget folder in your project:
+2. Go to the details widget folder in your project:
 
         cd ui/widgets/YOUR-ENTITY-NAME/detailsWidget
 
-3.  Then install and start your widget executing the command:
+3. Install and start your widget with the command:
 
         npm install && npm start
 
-4.  When the widget is started a browser window is opened with and the widget URL is loaded
+4. When the widget starts, a browser window opens, with the widget URL
 
-5.  If you’re not logged in you’re redirected to the login page.
+5. If you’re not logged in, you’re redirected to the login page
 
-6.  Log in using:
+6. Log in using:
 
         Username: user
         Password: user
 
-7.  You’ll be redirected to the widget page and you can see the widget form with the ID 1 loaded.
-
-### Widget Details notes:
+7.  Once logged in, you can see the widget details with the ID 1 loaded
 
-If you want to load other data you have to change the index.html file in the public folder:
+#### Widget Details notes:
 
-    cd ui/widgets/YOUR-ENTITY-NAME/detailsWidget/public
-
-and change the "id" attribute in this line:
+If you want to use custom data, edit the `index.html` file in the public folder ui/widgets/YOUR-ENTITY-NAME/detailsWidget/public. Change the "id" attribute in this line:
```

Comment:
Same as above, "If you want to view the details for a different row in the database..."